packages/svelte-undp-design/src/lib/Button/Button.svelte

```diff
@@ -6,13 +6,16 @@
 	export let title = '';
 	export let isArrow = false;
 	export let isPrimary = true;
+	export let isDisabled = false;
 
 	const handleClicked = () => {
-		dispatch('clicked');
+		if (!isDisabled) {
```

Comment:
@lrt-sibat please also add a changeset file for `svelte-undp-design` package. otherwise, it will not be released to NPM

Reply:
Hi Jin, a changeset file was already created [here](https://github.com/UNDP-Data/geohub/blob/feat/ceei-dashboard/.changeset/eleven-houses-occur.md)

Reply:
@lrt-sibat ok. I updated your changeset files